### PR TITLE
Fix docs copy paste syntax

### PR DESCRIPTION
**Pull request by swalker326**
### What's added in this PR?

- Fixed the copy paste of code snippets getting really strange spacing and breaks
- Added one liner to make the `package.json`s needed
- Re-worded some of the language around the chrome extension
- Added steps to properly build the apps in the correct order
- Fixed ordering issue with step number duplication

**Current live copy**

```bahs
npx nx 
add
 @nx/react
npx nx g @nx/react:host --remotes
=
remote1,remote2 --bundler
=
rspack --directory
=
apps/host
```

**New copy**
```bash
npx nx run-many -t build --parallel=1 --skip-nx-cache --verbose -p host remote1 remote2
# Or alternatively only serve host 
npx nx run host:serve
```

### (Required) Pre-PR/Merge checklist

- [ ] I have added/updated our feature to sync with this PR
- [x] I have added an explanation of my changes
- [ ] I have/will run tests, or ask for help to add test

### CODE REVIEW

**Comment by zackarychapple** [comment]
The alternative command should be a separate block or tab in the code block.  Otherwise it makes the copy button worthless.

**Comment by swalker326** [comment]
> The alternative command should be a separate block or tab in the code block. Otherwise it makes the copy button worthless.

I removed the alternative command in the one code block as it didn't work and the other was much more concise

File: docs/recipes/react-rspack-nx.mdx [hunk 1]
```diff
@@ -161,42 +165,52 @@ export default composePlugins(
...
-#### 6. Rollback to previous version 
+```bash Title=Terminal
+nx run remote1:build
+```
 
-The first option in the dropdown menu under **Version** is the latest deployment of your app. In the dropdown menu you will see more previous version of your app. 
+```bash Title=Terminal
+nx run remote2:build
+```
+Now you can serve the host application, which through nx magic will start the two remotes also
```
Comment (zmzlois) [inline docs/recipes/react-rspack-nx.mdx:181]:
```suggestion
Now you can serve the host application to check if everything is working properly, which through nx magic will start the two remotes. 
```

**Review by zmzlois — APPROVED** [review]

File: docs/recipes/react-rspack-nx.mdx [hunk 2]
```diff
@@ -122,7 +126,7 @@ In remote2's `package.json`, add below field:
 
 #### 4. Create your first commit 
 
-After making sure each application in this project has a `package.json` and the naming align with the unique name in module federation config, you will need to make sure this project is a github repository and has at least one commit hash by running `git commit -m "commit something"` command. 
+After making sure each application in this project has a `package.json` and the naming aligns with the unique name in module federation config, you will need to make sure this project is a github repository and has at least one commit hash by running `git commit -m "commit something"` command. 
 
 #### 5. Adding configuration for Zephyr
```
Comment (zmzlois) [inline docs/recipes/react-rspack-nx.mdx:132]:
```suggestion
In all the build configuration files, start with importing Zephyr's plugin: 

```ts title="Import Zephyr" 
import { withZephyr } from "zephyr-webpack-plugin" 
```

And then add the plugin under module federation plugin: 

```ts title="Add plugin" 
withZephyr()
```

**Pull request merged**